kvrocks.conf

```diff
@@ -547,7 +547,7 @@ rocksdb.row_cache_size 0
 # files opened are always kept open. You can estimate number of files based
 # on target_file_size_base and target_file_size_multiplier for level-based
 # compaction. For universal-style compaction, you can usually set it to -1.
-# Default: 4096
+# Default: 8096
```

Comment:
This value is 4096 by default: https://github.com/apache/kvrocks/blob/unstable/src/config/config.cc#L166